planner/core/collect_column_stats_usage.go

```diff
@@ -50,21 +69,21 @@ func (c *predicateColumnCollector) addPredicateColumn(col *expression.Column) {
 	}
 }
 
-func (c *predicateColumnCollector) addPredicateColumnsFromExpression(expr expression.Expression) {
+func (c *columnStatsUsageCollector) addPredicateColumnsFromExpression(expr expression.Expression) {
 	cols := expression.ExtractColumnsAndCorColumns(c.cols[:0], expr)
 	for _, col := range cols {
 		c.addPredicateColumn(col)
 	}
 }
 
-func (c *predicateColumnCollector) addPredicateColumnsFromExpressions(list []expression.Expression) {
+func (c *columnStatsUsageCollector) addPredicateColumnsFromExpressions(list []expression.Expression) {
```

Comment:
Can we merge this part? It seems the only difference is whether the expression is a slice or not.
Besides, we only need to export `expression.ExtractColumnsAndCorColumnsFromExpressions`, not all of them.

Reply:
Done.